makefile

```diff
@@ -22,13 +22,18 @@ ifeq ($(OS),Windows_NT)
 gocache := $(shell cygpath -m $(gocache))
 gopath := $(shell cygpath -m $(gopath))
 endif
-goEnv := GOPATH=$(gopath) GOCACHE=$(gocache)$(if $(GO_BIN_PATH), PATH="$(shell dirname $(GO_BIN_PATH)):$(PATH)")
+
+export GOPATH := $(gopath)
+export GOCACHE := $(gocache)
+ifneq ($(GO_BIN_PATH),)
+	export PATH := $(shell dirname $(GO_BIN_PATH)):$(shell printenv PATH)
+endif
```

Comment:
I'd rather not export, it makes it easier to see what happens by copying the (admitedly too long) shell line.

Reply:
I actually would rather export because I've found people are highly prone to not editing the makefile correctly. Whenever someone adds something, it's too easy to forget it or accidentally rely on the go environment without realizing it (e.g. gometalinter always relied on the "go" binary in the PATH), so global state that applies to the entire makefile seems preferable in this case.

Regardless, I re-added the goEnv if you think its' easier. However, I don't trust our ability to write/edit makefiles correctly since not everyone knows how the makefiles work.